### ISSUE TITLE
Exception when using env. variables for the http port property

### ISSUE THREAD

**Issue by tlvlp** [redhat-developer/intellij-quarkus#472]
The application.properties parser assumes that http port will only contain a parseable integer and throws the following exception.

application.properties
```properties
quarkus.http.port=${PORT:8081}
```

Exception:

```log
Error while converting '${PORT:8081}' as Integer for key 'quarkus.http.port'

java.lang.NumberFormatException: For input string: "${PORT:8081}"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:638)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at com.redhat.devtools.intellij.lsp4mp4ij.psi.internal.core.project.AbstractConfigSource.getPropertyAsInt(AbstractConfigSource.java:149)
	at com.redhat.devtools.intellij.lsp4mp4ij.psi.core.project.PsiMicroProfileProject.getPropertyAsInteger(PsiMicroProfileProject.java:81)
	at com.redhat.microprofile.psi.internal.quarkus.jaxrs.java.QuarkusJaxRsCodeLensParticipant.beginCodeLens(QuarkusJaxRsCodeLensParticipant.java:44)
	at com.redhat.devtools.intellij.lsp4mp4ij.psi.core.PropertiesManagerForJava.lambda$collectCodeLens$3(PropertiesManagerForJava.java:144)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at com.redhat.devtools.intellij.lsp4mp4ij.psi.core.PropertiesManagerForJava.collectCodeLens(PropertiesManagerForJava.java:144)
	at com.redhat.devtools.intellij.lsp4mp4ij.psi.core.PropertiesManagerForJava.lambda$codeLens$1(PropertiesManagerForJava.java:122)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:841)
	at com.redhat.devtools.intellij.lsp4mp4ij.psi.core.PropertiesManagerForJava.codeLens(PropertiesManagerForJava.java:115)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$getJavaCodelens$11(QuarkusLanguageClient.java:154)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$runAsBackground$2(QuarkusLanguageClient.java:103)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$runAsBackground$3(QuarkusLanguageClient.java:111)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1728)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

**Comment by tlvlp** [comment]
@jeffmaury I have also added another PR that might also solve the root cause of the exceptions.

**Comment by tlvlp** [comment]
@angelozerr do you think that this will be addressed in the near future?

**Comment by angelozerr** [comment]
@tlvlp sorry I was busy with the future Qute support. I will try to implement but I don't know if I could do that this week.

**Comment by angelozerr** [comment]
@jeffmaury @tlvlp please see the PR done for JDT https://github.com/eclipse/lsp4mp/pull/283/files and adapt it for IJ. It should take care of this issue and property expression.

**Comment by tlvlp** [comment]
Unfortunately since raising this issue, I no longer use Quarkus or the Quarkus IJ plugin, and I see that the related PR is now grossly outdated. So this is probably best to be picked up by someone from RedHat.

**Comment by jeffmaury** [comment]
Fixed by e7db52191364268252c46677b206f67898ed2574

**Comment by dxps** [comment]
That's great! No intention to put a pressure, but I'm wondering when this will be included in the next update of the plugin, please :question: 

Thanks!

Asking since currently I'm getting a notification bubble about it, at every couple of seconds, although I'm explicitly set it to no show it (I know, the latter is part of the new IntelliJ UI's feature, not related to this plugin).

![image](https://user-images.githubusercontent.com/14140226/215528721-43880d23-40aa-434a-ab87-e28ce317a3b1.png)
